cms/templatetags/filter_video_id.py

```diff
@@ -0,0 +1,24 @@
+"""Template filter for rendering Youtube videos"""
+import re
+
+from django import template
+from django.template.defaultfilters import stringfilter
+
+register = template.Library()
+
+
+@register.filter(is_safe=True)
+@stringfilter
+def get_video_id(url):
+    """
+    Get Youtube video url and return video_id.
+    """
+    regex = re.compile(
+        r"^(https?://)?(www\.)?(youtube\.com/watch\?v=)?(?P<id>[A-Za-z0-9\-=_]{11})"
```

Comment:
jumping here, any test for that one ?

Reply:
I thought about it initially, but can't find any example in our code to test template tags.

Reply:
please be sure this regex against it. 

> Valid:
>         'http://youtu.be/_lOT2p_FCvA',
>         'www.youtube.com/watch?v=_lOT2p_FCvA&feature=feedu',
>         'http://www.youtube.com/embed/_lOT2p_FCvA',
>         'http://www.youtube.com/v/_lOT2p_FCvA?version=3&amp;hl=en_US',
>         'https://www.youtube.com/watch?v=rTHlyTphWP0&index=6&list=PLjeDyYvG6-40qawYNR4juzvSOg-ezZ2a6',
>         'youtube.com/watch?v=_lOT2p_FCvA',
>         'https://www.youtube.com/watch?v=S6q41Rfltsk'
>       Invalid:
>         'youtu.be/watch?v=_lOT2p_FCvA'